server/routes/adjustmentRoutes.ts

```diff
@@ -81,14 +81,22 @@ export default class AdjustmentRoutes {
 
     let remandDecision
     let relevantRemand
+    let remandBannerVisible = false
     if (roles.includes('REMAND_IDENTIFIER')) {
       try {
         remandDecision = await this.identifyRemandPeriodsService.getRemandDecision(nomsId, username)
         relevantRemand = await this.identifyRemandPeriodsService.calculateRelevantRemand(nomsId, username)
+        remandBannerVisible = serviceDefinitions.services.adjustments.thingsToDo.things.some(
```

Comment:
Have you tested how this looks? Things to do aren't only present in adjustments, is a ccrd wide feature. This should really be in the backend, otherwise your going to see the thing to do banner in all places other than adjustments.